lib/ansible/utils/display.py

```diff
@@ -100,10 +99,9 @@ def get_text_width(text):
             w = 0
         width += w
 
-    if width == 0 and counter and not LOCALE_INITIALIZED:
+    if width == 0 and counter:
```

Comment:
Is there a string we can use to trigger this error condition?

Reply:
Technically we have a test for it, but per my comment in the units, I could never actually find a way to make it reliably happen on Python3.

https://github.com/ansible/ansible/blob/5e369604e1930b1a2e071fecd7ec5276ebd12cb1/test/units/utils/test_display.py#L38-L40

Maybe by the time I figure out how to properly break encoding detection I might have a way to enable it for Python3.

Reply:
I got those units working on Py3.